src/model/ZoneHVACTerminalUnitVariableRefrigerantFlow.cpp

```diff
@@ -814,7 +927,11 @@ namespace model {
   }
 
   HVACComponent ZoneHVACTerminalUnitVariableRefrigerantFlow::supplyAirFan() const {
-    return getImpl<detail::ZoneHVACTerminalUnitVariableRefrigerantFlow_Impl>()->supplyAirFan();
+    boost::optional<HVACComponent> fan_ = getImpl<detail::ZoneHVACTerminalUnitVariableRefrigerantFlow_Impl>()->supplyAirFan();
+    if (!fan_) {
+      LOG_AND_THROW(briefDescription() << " does not have an Supply Air Fan attached.");
+    }
+    return fan_.get();
```

Comment:
Since I didn't change the public side... I'm throwing if I can't find it on the public side. @kbenne